README.md

```diff
@@ -1,399 +1,394 @@
-# terraform-best-practices
-### Welcome! This repo contains a set of best practices to be followed when contributing to the Palo Alto Networks terraform modules used by Professional Services.
+# Terraform Best Practices
 
+Welcome! This repo contains a set of best practices to be followed when contributing to the Palo Alto Networks terraform modules used by Professional Services.
 
 ## Table of Contents
 
 1. [Versioning](#1-versioning)
 2. [Coding Practices](#2-coding-practices)
 3. [Tips and Tricks](#3-tips-and-tricks)
 4. [Terraform Module Structure](#4-Terraform-Module-Structure)
-5. [Document Generation](#5-Document-Generation)
-6. [Terraform Module Testing](#6-Terraform-Module-Testing)
+<!-- TODO 5. [Document Generation](#5-Document-Generation) -->
+<!-- TODO 6. [Terraform Module Testing](#6-Terraform-Module-Testing) -->
 
+## 1. Versioning
 
-## `1. Versioning`
+### 1.1 Terraform Version
 
-### `1.1 Terraform Version`
-* All terraform configuration should be written using the latest version of the Terraform GA release available (0.12.26 as of writing this guide). 
+All code should support the latest minor version of the latest Terraform GA release.
 
-  > Existing Terraform modules should be tested against the latest Terraform releases using a test methodology to be defined later. 
+<!-- TODO All code should work with the latest three(?) major Terraform releases, requiring all the critical tests to pass as specified in the [Testing](#6-Terraform-Module-Testing) section. Workarounds, if needed, are allowed if they are clearly documented. -->
 
-### `1.1 Module Version`
-* All terraform modules should be versioned using Git tags. Git tag name should adhere to the following naming convention 
+### 1.1 Module Version
+
+* All terraform modules should be versioned using Git tags. Git tag name should adhere to the following naming convention
 
 > Major.Minor.Patch => 0.1.0
 
-* Version should be pinned in the root module where the modules are called from using the `ref` argument 
+* Version should be pinned in the root module where the modules are called from using the `ref` argument:
 
       module "bootstrap_bucket" {
         source          = "spring.paloaltonetworks.com/mekanayake/terraform-aws-panfw-bootstrap?ref=0.1.0"
         bucket_prefix   = "GlobalProtect-bootstrap-"
```

Comment:
Is the trailing dash intended in "GlobalProtect-bootstrap-"? Just asking :)

Reply:
Not my code, but yes - it is intended.